frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestTimeline/ChangeRequestTimeline.tsx

```diff
@@ -62,36 +69,24 @@ export const determineColor = (
     changeRequestStateIndex: number,
     displayStage: ChangeRequestState,
     displayStageIndex: number,
-    failureReason?: string,
 ) => {
     if (changeRequestState === 'Cancelled') return 'grey';
 
     if (changeRequestState === 'Rejected')
         return displayStage === 'Rejected' ? 'error' : 'success';
     if (
         changeRequestStateIndex !== -1 &&
-        changeRequestStateIndex > displayStageIndex
+        changeRequestStateIndex >= displayStageIndex
     )
         return 'success';
-    if (
-        changeRequestStateIndex !== -1 &&
-        changeRequestStateIndex === displayStageIndex
-    ) {
-        return changeRequestState === 'Scheduled'
-            ? failureReason
-                ? 'error'
-                : 'warning'
-            : 'success';
-    }
```

Comment:
Scheduled change requests never reach this level anymore, so we can fold it into the previous one